src/ralph/assets/subscribers.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+import logging
+
+import pyhermes
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from ralph.assets.models.assets import Environment, Service, ServiceEnvironment
+from ralph.assets.models.base import BaseObject
+
+logger = logging.getLogger(__name__)
+
+
+def _update_service_owners(service, business_owners, technical_owners):
+    """
+    Update business and technical owners for selected service.
+
+    Args:
+        service: Service
+        business_owners: dict of owners
+        technical_owners: dict of owners
+    """
+    business_owners = [
+        get_user_model().objects.get(username=i['username'])
```

Comment:
Seriously?, previously I have get_or_create you wanted to change  to get() and I told you (face to face) about it that user will not exist.

Which version i have select?